airbyte-integrations/connectors/destination-s3-data-lake/src/main/kotlin/io/airbyte/integrations/destination/s3_data_lake/S3DataLakeDestination.kt

```diff
@@ -6,12 +6,16 @@ package io.airbyte.integrations.destination.s3_data_lake
 
 import io.airbyte.cdk.AirbyteDestinationRunner
 import io.airbyte.cdk.load.command.aws.AwsToolkitConstants
+import io.github.oshai.kotlinlogging.KotlinLogging
+
+private val logger = KotlinLogging.logger {}
 
 object S3DataLakeDestination {
     val additionalMicronautEnvs = listOf(AwsToolkitConstants.MICRONAUT_ENVIRONMENT)
 
     @JvmStatic
     fun main(args: Array<String>) {
+        logger.info { "EDGAO DEBUG" }
```

Comment:
Probably don't merge this.

Reply:
yep, this was just to make connectors CI run. reverted in 47c43af418